prov/gni/include/gnix_mr_cache.h

```diff
@@ -150,7 +151,7 @@ typedef struct gnix_mr_cache_attr {
 	void *reg_context;
 	void *dereg_context;
 	void *destruct_context;
-	struct gnix_mr_notifier *notifier;
+	struct gnix_smrn *smrn;
```

Comment:
I'll add an issue for this since it would include adding documentation for more than just this change.